### ISSUE TITLE
Project status

### ISSUE THREAD

**Issue by theredcat** [konobi/forge#4]
Hi,

I've found your project and you're currently doing what I'm looking to implement. I just want to know if you have working deployments, cause i can't get it up and running. 

The Intel PXE can get a DHCP IP address from Forge and then iPXE successfully load in my test VM (VBox + VBox Extension using Intel PXE) but at this point iPXE failed to dhcp. I can see the requests and (I think correct) responses on VM interfaces, but then iPXE seem to completely ignore the DHCP offers. 
![2015-05-10 01_43_12-pxe-test-client en fonction - oracle vm virtualbox](https://cloud.githubusercontent.com/assets/5129654/7552467/175d2c80-f6b6-11e4-9dad-169f57aa11ae.png)

**Comment by theredcat** [comment]
Server log : http://pastebin.com/vH6z6K9K

**Comment by konobi** [comment]
Hi there...

Using virtualbox was a problem for me as I couldn't get it to behave properly in terms of making it able to use the virtual network adapter in promiscuous mode. I've had better luck using VMWare, but I've also been developing on mac.

I was able to verify that messages were being sent to the VM, but just not being responded to by using WireShark to capture all the traffic on the virtual interface. Can you have a look at seeing what's happening over the "wire" and that the promiscuous mode is sticking?

thanks.

**Comment by theredcat** [comment]
Indeed, the problem here is vbox. I've recompiled iPXE with systematic debug in udp_rx and udp_tx functions and I can clearly see that I don't receive anything (UDP tx OK but UDP rx is never triggered) but I can see the DHCP offers with wireshark : http://www.filedropper.com/ipxeforge

This was blowing my mind so I've used my physical lab today and using the same VM server in vbox, but bridged to PHY network and PHY clients (Thinkpad T410 + Dell Pe2950 + Dell R200) everything is running fine.

This seem to be a iPXE issue with Vbox network since the Vbox Intel PXE code is working fine. Maybe in card driver.

I've spotted another issue in server. When the same client send two boot request in a short amount of time, the server crash. Using https://github.com/chipzz/node-dhcpd fixes the problem

**Comment by konobi** [comment]
Mmm... yeah, there could be dhcpd issues that I haven't been aware of, since issues on forks go to the original project.

I do have a todo task to get better logging built into node-dhcpd and forge, but it's still under development. I had a good testing scenario for forge until recently, but I may be able to try interacting with SDC as a decent scenario.

It looks like the upstream dhcpd repo is up to date, but perhaps it's not been released to npm.

**Comment by konobi** [comment]
@theredcat If you could join me on the gitter.im room at some point, it'd be useful to get some more specifics of issues that you're having. Also to get some more of your use cases.
